freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/freeipa/test/model/CheckUsersInGroupRequest.java

```diff
@@ -0,0 +1,40 @@
+package com.sequenceiq.freeipa.api.v1.freeipa.test.model;
+
+import java.util.Set;
+
+import javax.validation.constraints.NotEmpty;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.sequenceiq.service.api.doc.ModelDescriptions;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel("CheckUsersInGroupRequest")
```

Comment:
please add `v1` to the name of the requests as we do it with the others

Reply:
done